ReactAndroid/src/main/java/com/facebook/react/views/scroll/ReactHorizontalScrollView.java

```diff
@@ -1086,14 +1086,19 @@ private void flingAndSnap(int velocityX) {
       // try to manually interact with OverScroller instead
       // if velocity is 0 however, fling() won't work, so we want to use smoothScrollTo
       mActivelyScrolling = true;
-
+      int snapToVelocityX = targetOffset - getScrollX();
+      if (!hasCustomizedFlingAnimator) {
+        // The default animator requires boost on initial velocity as when snapping velocity can
+        // feel sluggish for slow swipes
+        snapToVelocityX = snapToVelocityX * 50;
```

Comment:
I'm not really a fan of Magic numbers. 
What if this change breaks other use cases where the users don't need the speed boost?

Also, this change affect everyone not using a customFlingAnimator. I wonder whether would be better to add a new props optional to this component in order to control the `snapToVelocity` from JS. 

WDYT?

Reply:
That's a fair ask. Let me put some effort into it this week. Thanks for the review.

Reply:
just a quick question, if i want to modify this file for example and go to this path in my 
project ../node_modules/react-native/.../ReactHorizontalScrollView.java 
then, to see the change i just have to build the project and it will take my change ? or are there any more actions needed ? like building from source or whatever

Reply:
Yea, you have to build from source, depending on how your build.gradle is structure you will have to do the following:

```
 configurations.all {
        resolutionStrategy.dependencySubstitution {
            substitute(module("com.facebook.react:react-native"))
                    .using(project(":ReactAndroid")).because("On New Architecture we're building React Native from source")
            // If using Hermes
            // substitute(module("com.facebook.react:hermes-engine"))
            //         .using(project(":ReactAndroid:hermes-engine"))
            //         .because("On New Architecture we're building Hermes from source")
        }
    }

```

Reply:
Thanks for replying,
Well, I spend 2-3 days trying building from source, and always bumped into different errors.
Now I'm trying modifying this class via [native module](https://reactnative.dev/docs/native-modules-android).
I don't fully understand yet how it actually works and if its possible but i will learn in the process.
I just want to make my scroll view (horizontal FlatList & paging enabled) to only listen to almost completely horizontal vector swipe in order to change page. also cut the fling event maybe, cause it is just to sensitive.